ver.bat

```diff
@@ -0,0 +1 @@
+cd ..\viper_client-master & python client.py -c verify -p 53743 & cd ..\viperserver
```

Comment:
You could parameterise this script and ter.bat s.t. they accept the port number as argument number 1.